src/lib.rs

```diff
@@ -96,6 +96,18 @@ impl<K, V> LruCache<K, V> where K: PartialOrd + Ord + Clone {
             self.map.insert(key, (value, time::SteadyTime::now()));
         }
     }
+/// Remove a key/value pair from cache
```

Comment:
Yea sorry Brian, I meant to merge earlier this makes perfect sense. Thanks a lot for the PR, nice one.